src/grt/test/congestion1.ok

```diff
@@ -59,128 +59,155 @@ metal10    Vertical         2256            47          97.92%
 [INFO GRT-0101] Running extra iterations to remove overflow.
 [INFO GRT-0124] Update congestion history type 1.
 [INFO GRT-0102] iteration 1, enlarge 20, costheight 6, threshold 10 via cost 2. 
-log_coef 0.84, healingTrigger 0 cost_step 2 L 1 cost_type 1 updatetype 1.
+log_coef 0.77, healingTrigger 0 cost_step 2 L 1 cost_type 1 updatetype 1.
```

Comment:
is this really in the grt output?? it does not look like anything useful to a users to me; it is all gibberish.
it should be a debugPrint or removed